unit-tests/py/rspy/log.py

```diff
@@ -4,6 +4,17 @@
 import sys
 
 
+def _write( s ):
+    """
+    When s is long, write() doesn't seem to work right and only part of the string gets written!
+    """
+    x = 0
+    chunk = 8192
+    while( x < len(s) ):
+        sys.stdout.write( s[x:x+chunk] )
```

Comment:
at the last chunk you need to use min(x+chunk, len(s) - x) no?

Reply:
```
>>> s = 'abc'
>>> s[2:200]
'c'
```